packages/client/src/settings/getConfiguration.ts

```diff
@@ -12,7 +12,7 @@ import type { GetConfiguration } from './types';
  *
  * @returns Promise that will resolve when the call to the endpoint finishes.
  */
-const getConfiguration: GetConfiguration = (code, query, config) =>
+const getConfiguration: GetConfiguration = (code, query = {}, config) =>
```

Comment:
Question: Is the default parameter value necessary? If not, you can remove it because they are not adding value.

Reply:
Removed